data/json/recipes/other/tool.json

```diff
@@ -55,6 +55,36 @@
     "tools": [ [ [ "swage", -1 ] ] ],
     "components": [ [ [ "scrap", 2 ] ] ]
   },
+  {
+    "result": "shears",
+    "type": "recipe",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_TOOLS",
+    "skill_used": "fabrication",
+    "difficulty": 3,
+    "time": "3 h",
+    "autolearn": true,
+    "reversible": true,
```

Comment:
Wouldn't make sense for this recipe to be reversible, I think. Breaking down shears back into metal would need at least a hacksaw and at most a forge; same for recipe below.